src/platforms/android/migration.mdx

```diff
@@ -6,6 +6,31 @@ redirect_from:
 description: "Migrating between versions of Sentry's SDK for Android."
 ---
 
+## Migrating from `io.sentry:sentry-android 5.x` to `io.sentry:sentry-android 6.0.0`
+
+* Kotlin plugin is upgraded to `1.5`.
+* Kotlin `languageVersion` is upgraded to `1.4`.
```

Comment:
The wording is a bit confusing. Does the user need to take action? Are we telling users that we updated the minimum required version? If not, maybe: "the sdks kotlin version minimum requirements is now 1.4" or something

Reply:
They don't need to do anything, but it's incompatible with Kotlin 1.3 now, 1.3 is deprecated anyway with Kotlin 1.5 which is upgraded in the item above.